packages/ecl/build.sh

```diff
@@ -0,0 +1,46 @@
+TERMUX_PKG_HOMEPAGE=https://common-lisp.net/project/ecl/
+TERMUX_PKG_DESCRIPTION="ECL (Embeddable Common Lisp) is an interpreter of the Common Lisp language."
```

Comment:
ecl is a compiler with runtime, not an interpreter

Reply:
Hey @dkochmanski, good to see you here :). I just copied the first sentence from https://common-lisp.net/project/ecl/main.html here :). Happy to adjust it if you think it's not correct.

Reply:
thanks, I need probably to rephrase it. If you are interested in details:
ecl always compiles the code (either with the bytecodes compiler or the c compiler), in the former case it executes the bytecode in a bytecode vm and that could be called interpreter.

Reply:
I guess it can be called an interpreter because python and most other scripting languages work the same way: compile to bytecode, then run that. ECL has the additional option of compiling to C.

Reply:
I disagree, interpretation has a well defined meaning and it does not fit the model of compile->bytecode->execute. But I don't care enough to insist (especially that our own webpage is wrong :-)